tests/wrap.py

```diff
@@ -34,25 +34,22 @@ def testMakeControl(self):
         config.foo = 2
         config.bar.append("baz")
         control = config.makeControl()
-        self.assert_(testLib.checkControl(control, config.foo, config.bar))
+        self.assert_(testLib.checkControl(control, config.foo, config.bar._list))
```

Comment:
Because the type `List` is not an actual Python list and thus cannot be passed to a C++ function accepting `std::vector`. We might put in a proposal to change pybind11 to accept any sequence as input for functions that take a `std::list`, `std::vector` or `std::tuple` but that might be a relatively big change.